src/components/Nav.vue

```diff
@@ -33,22 +33,35 @@
     >
 
       <div class="navbar-end">
-        <div class="navbar-item">
+        <div class="navbar-item" v-if="!getLog">
           <div class="buttons">
-            <router-link v-if="!getLog" to="/" class="button is-light">
+            <router-link to="/" class="button is-light">
               Se connecter
             </router-link>
-            <router-link v-if="getLog" to="/profile" class="button is-light">
-              Profile
-            </router-link>
-            <router-link v-if="getLog && hasUnconfiguredBackend" to="/create-account" class="button is-light">
+          </div>
+        </div>
+
+        <div class="navbar-item has-dropdown is-hoverable" v-if="getLog">
```

Comment:
I always have a dropdown now, is that intended ? I'm not sure it was requested.

Reply:
Intended, since we're about to add the map section, and we now have a proper profile page (on the Odoo instance), it seemed to me it would be worth it moving "Profile" and "Disconnect" into the user dropdown. We talked about it in another issue or PR IIRC.

Reply:
It was not requested and you rightfully asked so I asked back to mathieu and stephan and they agree on the change.